ch_4/beyond_ex_dictff/beyond_ex_dictdiff.py

```diff
@@ -0,0 +1,54 @@
+# Exericse 1:
+# d1 = {'d': 1, 'e': 3, 'g': 5}
+# d2 = {'d': 2, 'e': 4, 'h': 6 }
+# d3 = {'d': 5}
+
+# first iteration of ex 1 - function takes in 2 dictionaries
```

Comment:
The challenge I'm running into is having this first exercise taking in an unlimited number of dictionaries. Right now, it takes in two dictionaries, and it works. But I'm having a challenge taking in more than that.